cmd/caddy/main.go

```diff
@@ -30,7 +30,6 @@ package main
 
 import (
 	caddycmd "github.com/caddyserver/caddy/v2/cmd"
-
```

Comment:
I don't like this change, the space is intentional here, because this file is meant to be copied and edited by hand (as a manual way of building Caddy with plugins etc)

Reply:
ahhh, I think this is fixable :)

Reply:
Should be all set now :)